beacon-chain/BUILD.bazel

```diff
@@ -93,6 +93,12 @@ go_test(
     deps = ["@com_github_urfave_cli//:go_default_library"],
 )
 
+go_path(
```

Comment:
Can you add a comment about what this could be used for?

Reply:
A comment here or in the code, or readme?

Reply:
Here, just above this line of code so some future person doesn't remove it thinking that it is unused.